Packs/SafeNet_Trusted_Access/Scripts/STAPostProcessing/STAPostProcessing.py

```diff
@@ -0,0 +1,58 @@
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+import traceback
+
+''' STANDALONE FUNCTION '''
+
+
+# Get Incident Details.
+def get_incident_sta():
+    return demisto.incidents()[0]
+
+
+# Check if user is a member of High risk group.
+def check_user_exist_group_sta(args: Dict[str, Any]):
+    return demisto.executeCommand('sta-user-exist-group', {
+        "userName": args.get('safenettrustedaccessusername'),
+        "groupName": args.get('safenettrustedaccesshighriskgroup'),
+        "using": args.get('safenettrustedaccessinstancename'),
```

Comment:
If there is arguments for the script, please add it to yml file and readme.

Reply:
These are the fields and will rename variable with **field**.

Reply:
Done.